src/main/resources/com/uipath/uipathpackage/entries/job/DynamicallyEntry/help-user.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    The name of the user. This feature is available only on modern folders.
```

Comment:
Let's add (from Azure DevOps plugin):

`The name of the user. This feature is available only on modern folders. This should be a machine user, not an orchestrator user. For local users, the format should be MachineName\\UserName".`